proposals/field-keyword.md

```diff
@@ -662,4 +662,50 @@ class C
 
 For reference, fields for *primary constructor parameters* are generated in similar cases - see [sharplab.io](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA+ABADAAgwRgDoARASwEMBzAOwgGcAXUsOgbgFgAoLjAJhwDCACgjAAVjDAMcAM1IwANgBMAlFwDeXHNrwocAWSFrOOnJpOmdxGMACulQgEE6dGFAZC5ipTlJ0c1LYKCiocFtoAvlxR3JyMULZSOADKIuKS0l7KxuamGHqGxqa5ltrWdg7Oru6e8sq+/oHBoVo6MRFAA).
 
-**Recommendation**: The backing field is generated when `field` is used only in omitted calls to *conditional methods*.
\ No newline at end of file
+**Recommendation**: The backing field is generated when `field` is used only in omitted calls to *conditional methods*.
+
+### Require overriding both accessors
+
+An auto-implemented property that overrides a virtual read-write property is currently required to override both accessors. Should that requirement be extended to accessors that use `field`?
```

Comment:
yes.  i don't see that changing here.  we could always have a feature request to enable overring just one side of a proeprty.  but that's been a long standing request anyways, and i don't see how 'field' changes things here.  Is there a reason to add this open question (As it seems out of scope for field to me).

Reply:
I believe this was already proposed at https://github.com/dotnet/csharplang/blob/main/proposals/field-keyword.md#overrides and confirmed at https://github.com/dotnet/csharplang/blob/main/meetings/2022/LDM-2022-05-02.md#partial-overrides-of-virtual-properties.